### DISCUSSION TITLE
There are a bunch of useful functions in here

### DISCUSSION THREAD

**Discussion by borkdude** [borkdude/rewrite-edn#27] [Ideas]
https://github.com/lread/rewrite-cljc-playground/blob/master/src/rewrite_cljc/zip/seqz.cljc

- assoc
- get
- etc

We might already have those, but maybe implemented them in a sub-optimal way.

**Comment by lread** [comment]
One note: the rewrite-clj versions are not as formatting friendly as rewrite-edn's.
For example rewrite-edn will indent new assoc or assoc-in items. 
Rewrite-clj inserts only minimum whitespace to keep code valid.
